### DISCUSSION TITLE
Filecoin Lotus Simplification

### DISCUSSION THREAD

**Discussion by BHK-Industry** [filecoin-project/community#606] [Developers]
Filecoin has been gaining a lot of traction as a decentralized storage network, but for many users, the process of making deals and utilizing the network can be complex and confusing.

One of the main issues is the lack of a user-friendly interface for making deals. This has led to many users struggling to understand the different variables involved, such as the price, duration, and data size. Additionally, the process of actually making a deal can be confusing and time-consuming for many users.

To simplify the Filecoin network for deal making, there are a few steps that can be taken. Firstly, a more intuitive and user-friendly interface can be developed. This should include clear and concise explanations of all the different variables involved in a deal, as well as a step-by-step process for making a deal.

Another way to simplify the Filecoin network is to streamline the process of making a deal. This could include automating some of the steps, such as setting the price and duration of a deal. Additionally, a feature that allows users to compare different deals based on their specific needs, such as data size and price, could be added to make it easier for users to find the best deal for them.

Finally, it would be beneficial to have a dedicated customer support team that can help users navigate the Filecoin network and answer any questions they may have. This would go a long way in reducing the confusion and frustration many users experience when trying to make a deal on the network.

By taking these steps, the Filecoin network can be made much more accessible and user-friendly for all users, making it easier for them to take advantage of its many benefits.

**Comment by steven004** [comment]
The suggestions here are way beyond Lotus implementation. Remember that Lotus is one of Filecoin client implementations, there are other implementations the users can take, such as Venus, Forest. And, for Lotus itself, the Boost (for the market implementation) was separated as an independent project. 

Filecoin is a multiple layer design, Lotus/Venus/Forest are on the lower layer to provide infrastructure support, including, chain services, consensus, storage proving, virtual machine etc. Storage/Retrieval market and deal making could be implemented in different ways, now we have Boost and Venus-Market to support it, and, it is expected there will be market supporting to simplify and automate the deal process via smart contracts or 3rd-part applications/services.
